manifests/charts/templates/configmap.yaml

```diff
@@ -171,27 +157,19 @@ data:
         },
         "pytorch": {
           "v1" : {
-            "image": "{{ .Values.kserve.servingruntime.pytorch.v1.image }}",
-            "defaultImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v1.defaultVersion }}",
-            "defaultGpuImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v1.defaultGpuVersion }}",
             "supportedFrameworks": [
               "pytorch"
             ],
             "multiModelServer": false
           },
           "v2" : {
-            "image": "{{ .Values.kserve.servingruntime.pytorch.v2.image }}",
-            "defaultImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v2.defaultVersion }}",
-            "defaultGpuImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v2.defaultGpuVersion }}",
```

Comment:
pytorch no longer need to differentiate between v1 and v2 as it is all using torchserve now, see [here](https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml#L49)

Reply:
yes, @yuzisun .. I updated.